### ISSUE TITLE
[Errata] Chapter 2, Question 3

### ISSUE THREAD

**Issue by kaoscoach** [awsccp/awsccp.github.io#6]
**Please describe the error you're reporting**
Appendix A, Answers to Review Questions, Chapter 2, Question 3. says B, D. The API calls/month and ECR free storage are available only under the Free Tier.

**Please indicate the chapter or appendix, section, and page number where the error appears (if applicable)**
Appendix A, Answers to Review Questions, Chapter 2, Question 3.

**Please include any other comments or questions**
In Chapter 2, Question 3 lists the answers as:

A. One million API calls/month on Amazon API Gateway
B. 10 GB of data retrievals from Amazon Glacier per month
C. 500 MB/month of free storage on the Amazon Elastic Container Registry (ECR)
D. 10 custom monitoring metrics and 10 alarms on Amazon CloudWatch

Therefore Appendix A, Answers to Review Questions, Chapter 2, Question 3. should be A, C. and not B,D.

**Comment by dbclinton** [comment]
Hi,
Thanks for sending this. I took another look at this and the original question was 

> "Which of the following usage will always be cost-free even after your account’s Free Tier has expired? (Select TWO.)"

It's looking for resource usage that will still be free even outside of the Free Tier. In that context, I think  B and D would be correct answers, since the Glacier and CloudWatch usage remain free, while API Gateway and ECR do now.
David

**Comment by kaoscoach** [comment]
I agree with your answer. Thanks for the clarification. I misunderstood the mentioning of "available only under the Free Tier. 

Can you check another answer? Chapter 4, Question 4. It asks, "Which of the following are _not_ globally based AWS services? (Select TWO.). 
A.	RDS
B.	Route 53
C.	EC2
D.	CloudFront

The answer says, 4.	B, D. Relational Database Service (RDS) and EC2 both use resources that can exist in only one Region. Route 53 and CloudFront are truly global services in that they’re not located in or restricted to any single AWS Region.

If B and D are global, how are they "_not_ global" with respect to the question?

**Comment by dbclinton** [comment]
>  I misunderstood the mentioning of "available only under the Free Tier.

The wording is a bit tricky. I had to read it very carefully myself (and I created that question).

> Can you check another answer? Chapter 4, Question 4.

That one is indeed wrong. The correct answers are, in fact, A and C. We've already submitted it for correction.
Thanks!
